opensearch-api/spec/opensearch/api/actions/cat/master_spec.rb

```diff
@@ -26,6 +26,7 @@
 
 require 'spec_helper'
 
+# <b>DEPRECATED:</b> Please use <tt>cluster_manager</tt> instead.
```

Comment:
I dont think we need to mark a test as deprecated since it's not going to be used or referenced by the end users.

Reply:
Yep, make sense. Removed deprecated tag.